src/NzbDrone.Core/Parser/Model/ImportListItemInfo.cs

```diff
@@ -10,6 +10,8 @@ public class ImportListItemInfo
         public string AuthorGoodreadsId { get; set; }
         public string Book { get; set; }
         public string BookGoodreadsId { get; set; }
+        public string BookIsbn10 { get; set; }
+        public string BookIsbn13 { get; set; }
```

Comment:
These are fine to add and set to from the list, but they need to be used to map the ListItems to good reads somehow, otherwise there is no real need for having them.

Reply:
Ended up removing these as creating a mapping would add another level of complexity